addons/hr_holidays/models/res_partner.py

```diff
@@ -17,7 +17,7 @@ def _compute_im_status(self):
                     partner.im_status = 'leave_online'
                 elif partner.im_status == 'away':
                     partner.im_status = 'leave_away'
-                else:
+                elif partner.im_status == 'offline':
```

Comment:
why that change?

Reply:
Ok makes sense! One last question, did you check if this PR should be forward-ported? There was a rework of the presence in 18, I don't know if it fixed the issue

Reply:
@Bertrand2 Yup, the same thing happens and the conflicting computes are still there